libunsf.c

```diff
@@ -1082,7 +1069,7 @@ static int unsf_mkdir(char *dir) {
 static void make_directories(UnSF_Options *options, SampleBank *sample_bank) {
     int i, tonebank_count = 0;
     char tmpname[80];
-    char *directory = NULL;
+    char directory[1024];
```

Comment:
Do we really want a path character limit of 1024? I wanted to move away from arbitrary boundaries. :/